tools/brl_checks.c

```diff
@@ -1,20 +1,24 @@
 /* liblouis Braille Translation and Back-Translation Library
 
-   Copyright (C) 2008 Eitan Isaacson <eitan@ascender.com>
```

Comment:
Maybe if we change all the headers to
```c
/*
 *
 *
```
this won't happen again by accident.

Reply:
Ah, sorry. I just removed the indent because clang-format would always break the long sbs copyright onto the next line. And that was kinda bugging me. But I guess it's not that important

Reply:
Ah, because now it's 92 wide and minus the 3 indent it fits exactly? :)